aca_required_temp.py

```diff
@@ -0,0 +1,189 @@
+
+import numpy as np
+
+import agasc
+from Chandra.Time import DateTime
+from Ska.Sun import nominal_roll, pitch
```

Comment:
Better not to overload common names like `pitch` which is prone to error (or forces using a different name when `pitch` (meaning the pitch value) would be appropriate.  Having `Ska.Sun.pitch(..)` in the code is fine and even a bit more clear.